index.bs

```diff
@@ -177,10 +177,11 @@ timestamp.
 <dfn>AudioPacket</dfn> contains decoded audio data. It will provide an
 {{AudioBuffer}} for rendering via {{AudioWorklet}}.
 
-A <dfn>VideoFrame</dfn> contains decoded video data. It will provide an
-{{ImageBitmap}} for rendering to {{Canvas}}. It should be possible to manipulate
-the {{ImageBitmap}} in {{WebGL}} and render the {{VideoFrame}} into a {{WebGL}}
-texture with texImage2D. It should also provide access to YUV data.
+A <dfn>VideoFrame</dfn> contains decoded video data. It can be drawn
+into {{Canvas}} with {{drawImage()}} or rendered into WebGL texture with
+{{texImage2D()}}.
+
+Issue: [ttoivone] shall access to raw YUV/RGB data be also allowed?
```

Comment:
Better to link to this issue -
https://github.com/WICG/web-codecs/issues/45

Reply:
Good point, I added the link. The issue #45 is actually reverse, creating the VideoFrame from YUV data while I was thinking about accessing a VideoFrame as an YUV data instead. But I'll update the issue.